kubernetes/resource_kubernetes_deployment_test.go

```diff
@@ -2402,6 +2412,24 @@ func testAccKubernetesDeploymentConfig_regression(provider, name string) string
               cpu    = "50m"
             }
           }
+          env {
+            name = "LIMITS_CPU"
+            value_from {
+              resource_field_ref {
+                container_name = "containername"
+                resource       = "requests.cpu"
+              }
+            }
+          }
+          env {
+           name = "LIMITS_MEM"
+            value_from {
+              resource_field_ref {
+                container_name = "containername"
+                resource       = "requests.memory"
```

Comment:
I added this field to the regression test to ensure that it wasn't going to overwrite existing deployments that use `resource_field_ref`. (`divisor` is intentionally omitted, because it doesn't exist in the released version of the kubernetes provider, so I can't test it here yet).